src/routes/+page.svelte

```diff
@@ -2,36 +2,35 @@
   import {
     storeBlockNumber,
     storeConnected,
+    storeMsaInfo,
+    storeToken,
     storeValidAccounts,
     transactionSigningAddress,
-    storeProviderId,
-    storeToken,
   } from '$lib/stores';
   import Connect from '$components/Connect.svelte';
   import Capacity from '$components/Capacity.svelte';
   import Intro from "$components/Intro.svelte";
   import Provider from '$components/Provider.svelte';
-  import AddControlKey from '$components/AddControlKey.svelte';
   import KeySelection from '$components/KeySelection.svelte';
   import {base} from "$app/paths";
```

Comment:
`npm run test` -> `Error: Failed to resolve import "$app/paths" from "src/routes/+page.svelte". Does the file exist?`

Reply:
- Also lint came in from his branch too and I will 'please the linter' in the next commit for this branch.